### ISSUE TITLE
macOS: Transparent items not visible (navicube, planes, rotation center)

### ISSUE THREAD

**Issue by maxwxyz** [FreeCAD/FreeCAD#15400]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

On macOS it seems that all transparent UI objects are not legible or even not recognizable, like the navicube, rotation center indicator, Draft grid, planes, origins,...
![Bildschirmfoto 2024-07-13 um 13 37 50](https://github.com/user-attachments/assets/4f943151-8261-4154-8dfe-50f45e91ed52)
![Bildschirmfoto 2024-07-13 um 13 38 07](https://github.com/user-attachments/assets/59072046-f6e8-4d68-afa2-ba79dcabba72)

![Bildschirmfoto 2024-07-13 um 13 44 43](https://github.com/user-attachments/assets/e50c5edd-4303-4268-9e8b-c2fe2a0cf2a3)

It is more legible with the dark theme and classic theme but also way worse than on Windows.
Light theme is not usable on macOS like this @kadet1090 @MisterMakerNL FYI

### Full version info

```shell
OS: macOS 13.6.7
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.38127 (Git)
Build type: Release
Branch: main
Hash: 9868533d8d0ef1f569cbde538b7a31350db503bd
Python 3.11.9, Qt 5.15.13, Coin 4.0.2, Vtk 9.2.6, OCC 7.7.2
Locale: C/Default (C)
Installed mods: 
  * freecad.gears 1.2.0
  * OpenTheme 2024.7.3
  * CfdOF 1.27.4
  * Defeaturing 1.2.2
  * Manipulator 1.5.7
  * fasteners 0.5.24
  * CurvedShapes 1.0.9
  * sheetmetal 0.4.22
  * Curves 0.6.41
```

### Subproject(s) affected?

Core

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by maxwxyz** [comment]
hovering the navicube improves the situation temporarily: 
![Bildschirmfoto 2024-07-13 um 13 41 52](https://github.com/user-attachments/assets/38d16e79-e2a7-4519-9ca4-3060e8ce5025)

With OpenLight the contrast is a bit better but still worse than on Windows:
![grafik](https://github.com/user-attachments/assets/835519b8-f4e7-40ae-bd0b-c85b4d972668)

Rotation indicator also not visible if you have no object behind it:
![grafik](https://github.com/user-attachments/assets/d6c70051-1e8a-471a-8777-c2c5b13a1b7c)

**Comment by luzpaz** [comment]
@johnocfii any chance you can test to confirm on macOS ?

**Comment by JohnOCFII** [comment]
> @JohnOCFII any chance you can test to confirm on macOS ?

Can confirm.  

<img width="1621" alt="Screenshot 2024-08-24 at 4 25 06 PM" src="https://github.com/user-attachments/assets/fc63150a-66a0-4ac8-b18a-e2fd5f620763">

**Comment by luzpaz** [comment]
@chennes this is kind of big deal for macOS, should it be a blocker ?

**Comment by chennes** [comment]
It makes the light theme difficult to use, but Classic is effectively unaffected. IMO it does not block the release.

**Comment by maxwxyz** [comment]
@chennes it's the same for all themes on my system, here is classic:
![Bildschirmfoto 2024-08-25 um 16 05 45](https://github.com/user-attachments/assets/52186824-faa7-42e7-b485-c454fea8529d)
I mean still better than light but worse than Windows

**Comment by chennes** [comment]
True, but the Navicube labels not showing is not serious enough, in my opinion, to warrant blocking the release. Is there any hint of a possible fix?

**Comment by MisterMakerNL** [comment]
@chennes Switching Qstyle probably.

**Comment by tomverbeure** [comment]
My 2 cents: I was ready to throw out FreeCAD as unusable on a Mac.

It took me half an hour to figure out that I should have switched from FreeCAD Light to FreeCAD classic, and that's only because I know GitHub well enough to go to the issues, filter on "MacOS" and find this specific issue.

**Comment by MisterMakerNL** [comment]
@tomverbeure which Freecad version are you using?

**Comment by maxwxyz** [comment]
@hyarion ^ the auto light mode on mac could lead to these issues

**Comment by tomverbeure** [comment]
> @tomverbeure which Freecad version are you using?

I just downloaded 1.0RC4. I had no FreeCAD preferences installed. The initial startup screen only shows FreeCAD Light and FreeCAD Dark as theme options, so I choose Light. I didn't even know a FreeCAD Classic theme existed.

**Comment by MisterMakerNL** [comment]
@tomverbeure We have only a handful of people here testing on a Mac. So you could really help out if you want.

**Comment by tomverbeure** [comment]
Feel free to ping me to try test builds. Not sure if I can help with anything else.

**Comment by hyarion** [comment]
This issue is available on macOS when using Light Theme on (System) Light Mode on 38871 too which does not have the auto-theme selection. So it's an old issue.

While this is clearly an issue, I don't think it's a blocker (should be fixed in 1.0.1)

I've also tested on RC4:
<details><summary>macOS + RC4</summary>
<p>

```
OS: macOS 14.5
Architecture: arm64
Version: 1.0.0RC4.39104 (Git) Conda
Build type: Release
Branch: (HEAD detached at 1.0rc4)
Hash: 83646753e051eaf438c0d5e9351cb57bd27377ab
Python 3.11.10, Qt 5.15.15, Coin 4.0.3, Vtk 9.3.0, OCC 7.8.1
Locale: C/Default (C)
Stylesheet/Theme/QtStyle: FreeCAD Light.qss/FreeCAD Light/Fusion
```

</p>
</details> 

Light theme in dark mode:
<img width="83" alt="image" src="https://github.com/user-attachments/assets/ece8253a-fdfb-45b6-b808-46d6a1acd6b3">

Light theme in light mode:
<img width="103" alt="image" src="https://github.com/user-attachments/assets/f6108c74-03c3-4a02-86e4-e1c9e5cf52e8">

Maybe changing the opacity to 75% when inactive? (Display > Navigation > Navigation Cube > Opacity when inactive)
Light theme in light mode with that adjusted:
<img width="99" alt="image" src="https://github.com/user-attachments/assets/0f6a8c13-8b1f-40e5-8245-a33b17dbdd2c">
<img width="184" alt="image" src="https://github.com/user-attachments/assets/313f72cc-8c55-4da1-8d38-8cb3c01a36cc">

**Comment by BanditoB1** [comment]
I just noticed the same thing.  Unless you are hovering over the Navigation Cube in MacOS on ARM, the labels for top, bottom, etc. do not show up.

As another Mac user, if you need someone to help test this out, please let me know.

**Comment by maxwxyz** [comment]
seems to be fixed in 1.1
